packages/qortal_hub.rb

```diff
@@ -0,0 +1,40 @@
+require 'package'
+
+class Qortal_hub < Package
+  description 'Desktop interface for Qortal'
+  homepage 'https://qortal.dev/'
+  version '0.5.2'
+  license 'Unknown'
```

Comment:
```suggestion
  license 'MIT'
```

https://github.com/Qortal/Qortal-Hub/blob/feature/initial-conversion/electron/package.json#L24

Reply:
That is Electron only.  There is also Android code too.